docs/_docs/Developers/build.md

```diff
@@ -111,6 +161,64 @@ application.name = Phoenicis PlayOnBSD
 ```
 Note : As there is not FreeBSD wine at the moment, the linux wine package will be displayed in the engine window.
 
+---
+
+## Linux From Source
+*Updated by [kreyren](https://github.com/Kreyren) at 24.02.2019*
```

Comment:
Why? It's not clear in which cases this should be used. Also: why do we need to explain OpenJFX compilation here? Don't they have a description themselves?

Reply:
@plata Since you have manuall compilation of java 11 in ubuntu if the distro doesn't support it. There might be other distros with simmilar problem and we can refference LFS.

Reply:
That is not the point. For an explanation how OpenJDK can be built you should link to the official build instructions. These instructions are not a part of Phoenicis but a part of OpenJDK.

Reply:
@madoar fair enough

Reply:
Fixed in https://github.com/PhoenicisOrg/phoenicis/pull/1860/commits/de38cb644be73ada5a681cb29cb400e4859958ce